tests/shadow_price.rs

```diff
@@ -0,0 +1,74 @@
+#[allow(clippy::float_cmp)]
```

Comment:
Not a good idea 😬 . We already have float_eq in the devdependencies, you should use that: https://docs.rs/float_eq/0.5.0/float_eq/macro.assert_float_eq.html